scripts/test

```diff
@@ -20,35 +18,14 @@ if [ "${BASH_SOURCE[0]}" = "${0}" ]; then
         usage
     else
         echo "Linting Bash scripts"
-        if [[ $CI ]]; then
```

Comment:
This was a hack to do a volume mounts on CircleCI so we could use the `shellcheck` container image. We don't need to do this on Jenkins anymore.